R/trust_types.R

```diff
@@ -0,0 +1,37 @@
+# This script joins the trust type (from the CQC rating data) onto the open trust data in the geographr package  
+
+# Load libraries 
+library(geographr)
+library(sf)
+library(tidyverse)
+library(readODS)
+
+source("R/utils.R") # for download_file()
+
+# Downloading CQC rating data as has information on what is the primary type of care trust provides 
+# Landing page of data: https://www.cqc.org.uk/about-us/transparency/using-cqc-data#directory
+tf <- download_file("https://www.cqc.org.uk/sites/default/files/04_January_2022_Latest_ratings.ods", "ods")
+
+raw_providers <-
+  read_ods(
+    tf,
+    sheet = "Providers",
+  )
+
+providers <- raw_providers |>
+  distinct(`Provider ID`, `Provider Primary Inspection Category`) 
+
+# Joining onto open trusts geographr data
+open_trusts <-
+  geographr::points_nhs_trusts |>
```

Comment:
```suggestion
  points_nhs_trusts |>
```
I think a choice needs to be made between either: (i) loading all objects from a package into a namespace using `library()`  calls, or calling specific objects directly from a package using `::`, rather than mixing both styles. While there will need to be some flexibility with this rule, I recommend we use the former option for this project where possible.